[Merged by Bors] - docs(install/*): put extra emphasis (#2439)
leanprover-community/mathlib: docs/install/debian.md

wget -q https://raw.githubusercontent.com/leanprover-community/mathlib-tools/master/scripts/install_debian.sh && bash install_debian.sh ; rm -f install_debian.sh && source ~/.profile
```

> ## What's next
> 
> :point_right: You should now read instructions about creating and working on [Lean projects](project.md)

Reviewer: ```suggestion
## You're not done yet!

:point_right: You should now read the instructions about creating and working on [Lean projects](project.md).
```
